events/tracks/gta2024-ml.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: event-track
+
+title: Mashine Learning
+description: Learn all about 
```

Comment:
Add description: `This module will introduce you to the world of machine learning using Galaxy`